Horizon/Horizon/Sources/Features/Account/AccountViewModel.swift

```diff
@@ -61,6 +61,7 @@ final class AccountViewModel {
 
         confirmLogoutViewModel.userConfirmation()
             .sink {
+                DomainJWTService.shared.clear()
                 sessionInteractor.logout()
```

Comment:
I think we should rather move this clear login to `sessionInteractor.logout()` so the view model doesn't hold business logic.